static/js/message_store.js

```diff
@@ -60,6 +62,11 @@ export function get_pm_full_names(message) {
                 blueslip.error("Unknown user id " + user_id);
                 return "?";
             }
+
+            if (muting.is_user_muted(user_id)) {
+                return $t({defaultMessage: "Muted user"});
+            }
+
             return person.full_name;
```

Comment:
Maybe we should move this internal function to be a `people.get_display_full_name_by_user_id(user_id)` that we can call from this and the reactions code path?  I think this function ends up being identical to the `reactions` code path one aside from nicer error-handling here.